api/src/main/java/io/kafbat/ui/service/StatisticsService.java

```diff
@@ -71,7 +95,11 @@ private Statistics createStats(ClusterDescription description,
                 Collectors.toMap(KafkaConnectState::getName, c -> c)
             )
         )
-        .build();
+        .controller(quorumInfo.isPresent() ? ClusterController.KRAFT : ClusterController.ZOOKEEPER);
+
+    quorumInfo.ifPresent(i -> stats.quorumLeaderId(i.leaderId()));
```

Comment:
I would save quoruminfo into stats, just in sake of future.

Reply:
Was my initial idea too, but I decided not to, as `QuorumInfo.nodes` is an empty collection for some reason, `voters` was non-empty but quite boring from a data perspective, the rest is just not useful in any way in current implementation. What do you think?

Reply:
I would still keep it there